packages/suite/src/views/wallet/coinmarket/savings/overview/components/WaitingForFirstPayment/index.tsx

```diff
@@ -24,14 +25,28 @@ const Description = styled.div`
     line-height: 18px;
 `;
 
-const WaitingForFirstPayment = () => (
-    <StyledCard>
-        <Header>
-            <Translation id="TR_SAVINGS_OVERVIEW_WAITING_FOR_FIRST_PAYMENT_HEADER" />
-        </Header>
-        <Description>
-            <Translation id="TR_SAVINGS_OVERVIEW_WAITING_FOR_FIRST_PAYMENT_DESCRIPTION" />
-        </Description>
-    </StyledCard>
-);
-export default WaitingForFirstPayment;
+interface WaitingForFirstPaymentProps {
+    paymentMethod?: SavingsPaymentMethod;
+    providerName?: string;
+}
+
+export const WaitingForFirstPayment = (props: WaitingForFirstPaymentProps) => {
+    const paymentMethod = props.paymentMethod?.toUpperCase() as Uppercase<SavingsPaymentMethod>;
+
+    return (
+        <StyledCard>
+            <Header>
+                <Translation
+                    id={`TR_SAVINGS_OVERVIEW_WAITING_FOR_FIRST_${paymentMethod}_PAYMENT_HEADER`}
```

Comment:
Is it a good idea to build a translation string on the go? I think it can be easily forgotten when you add new payment method

Reply:
It's already done elsewhere. The worst case scenario is that we will see missing translation message in the UI that will most likely be discovered during testing. AFAIK still better that other alternatives. Also my IDE was complaining about missing translation until I added one.